### PR TITLE
adding Oracle to Distribution table

### PR DESCRIPTION
**Pull request by jbottum**
Adding Oracle to table

### CODE REVIEW

File: content/en/docs/started/installing-kubeflow.md [hunk 1]
```diff
@@ -122,6 +122,13 @@ Packaged distributions are developed and supported by their respective maintaine
         <td><a href="/docs/distributions/charmed/">Docs</a></td>
         <td><a href="https://charmed-kubeflow.io/docs/quickstart">External Website</a></td>
       </tr>
+        <td>Kubeflow for Oracle Container Engine for Kubernetes</td>
+        <td>Oracle</td>
+        <td>Oracle Cloud Infrastructure (OCI)</td>
+        <td>1.6</td>
+        <td>N/A</td>
+        <td><a href="https://github.com/oracle-devrel/kubeflow-oke">External Website</a></td>
+
```
Comment (annajung) [inline content/en/docs/started/installing-kubeflow.md:131]:
```suggestion
      </tr>
      <tr>
        <td>Kubeflow for Oracle Container Engine for Kubernetes</td>
        <td>Oracle</td>
        <td>Oracle Cloud Infrastructure (OCI)</td>
        <td>1.6</td>
        <td>N/A</td>
        <td><a href="https://github.com/oracle-devrel/kubeflow-oke">External Website</a></td>
      </tr>
```

Comment (annajung) [reply content/en/docs/started/installing-kubeflow.md:131]: preview https://deploy-preview-3353--competent-brattain-de2d6d.netlify.app/docs/started/installing-kubeflow/ shows syntax issues, you need to wrap <td>'s in a <tr>

**Comment by annajung** [comment]
preview https://deploy-preview-3353--competent-brattain-de2d6d.netlify.app/docs/started/installing-kubeflow/

**Comment by jbottum** [comment]
@zijianjoy @chensun special request - would one of you please approve this so that we don't have to cherry pick this for the kubeflow docs ?  thanks!

File: content/en/docs/started/installing-kubeflow.md [hunk 2]
```diff
@@ -122,6 +122,14 @@ Packaged distributions are developed and supported by their respective maintaine
         <td><a href="/docs/distributions/charmed/">Docs</a></td>
         <td><a href="https://charmed-kubeflow.io/docs/quickstart">External Website</a></td>
       </tr>
+      <tr>
+        <td>Kubeflow for Oracle Container Engine for Kubernetes</td>
+        <td>Oracle</td>
+        <td>Oracle Cloud Infrastructure (OCI)</td>
+        <td>1.6</td>
+        <td>N/A</td>
+        <td><a href="https://github.com/oracle-devrel/kubeflow-oke">External Website</a></td>
```
Comment (zijianjoy) [inline content/en/docs/started/installing-kubeflow.md:131]: Brand guideline: We shouldn't call something as `Kubeflow for XXX` 
oracle

Suggest to rename it to `Oracle Container Engine for Kubeflow` or `Kubeflow on Oracle Container Engine distribution`.

Comment (julioo) [reply content/en/docs/started/installing-kubeflow.md:131]: Ok I update the title

**Comment by zijianjoy** [comment]
/lgtm
/approve

Thank you for the update!

**Pull request merged**